public/js/controllers/aside-controller.js

```diff
@@ -0,0 +1,369 @@
+var app=angular.module("myApp.controllers.aside",['ngRoute','ngAnimate'])
+    .controller("asideCtr", function($scope,$location, $rootScope) {
```

Comment:
You should not use socket.io like this. Please remove the line of code var socket=io.connect() and add it as a library in the controller header.